package.json

```
@@ -41,37 +41,37 @@
   "peerDependencies": {},
   "optionalDependencies": {},
   "dependencies": {
-    "class-validator": "0.13.1",
-    "express": "4.17.1",
+    "class-validator": "0.14.0",
+    "express": "4.18.2",
     "lodash": "4.17.21",
-    "nestjs-typeorm-paginate": "2.2.6",
-    "typescript": "4.4.4"
+    "nestjs-typeorm-paginate": "4.0.3",
+    "typescript": "4.9.4"
   },
   "peerDependency": {
-    "typeorm": "^0.2.30",
-    "@nestjs/common": "^8.1.2"
+    "typeorm": "^0.3.11",
+    "@nestjs/common": "^9.2.1"
   },
   "devDependencies": {
-    "@nestjs/common": "8.1.2",
-    "@nestjs/core": "8.1.2",
-    "@types/express": "4.17.13",
-    "@types/jest": "27.0.2",
-    "@types/lodash": "^4.14.176",
-    "@types/node": "^14.17.32",
-    "@typescript-eslint/eslint-plugin": "^5.3.0",
-    "@typescript-eslint/parser": "^5.3.0",
-    "eslint": "^8.1.0",
-    "eslint-config-prettier": "^8.3.0",
-    "husky": "^7.0.4",
-    "jest": "^27.3.1",
-    "lint-staged": "^11.2.6",
-    "node-mocks-http": "^1.11.0",
-    "prettier": "^2.4.1",
+    "@nestjs/common": "9.2.1",
+    "@nestjs/core": "9.2.1",
```

We have only two imports that rely on these packages as devDependencies in tests:

```
import { ExecutionContextHost } from '@nestjs/core/helpers/execution-context-host';
import { ROUTE_ARGS_METADATA } from '@nestjs/common/constants';
```

at a glance, both imports should be ok to use across versions of NestJS stretching back to 7 and perhaps earlier on (I'm particularly interested in v7 for opportunistic reasons aka its use in MarxanCloud 🙄), so I would suggest to keep these dependencies open to earlier NestJS versions as above with the peer-dep

```suggestion
    "@nestjs/common": "^7.0.0 || ^8.0.0 || ^9.0.",
    "@nestjs/core": "^7.0.0 || ^8.0.0 || ^9.0.",
```